corehq/apps/userreports/reports/specs.py

```diff
@@ -216,7 +218,7 @@ def aggregations(self, data_source_config, lang):
             aggregation = aggregation.order('_term', order=order)
         else:
             aggregation = ES_AGG_MAP[self.aggregation](self.column_id, self.field)
-        return filter(None, [aggregation])
```

Comment:
I kind of prefer the filter syntax for this, but understand that py3 changes it's behavior. would it be worth using six.moves.filter when possible? (also sucks that we won't be able to lint for this)

Reply:
There are a few alternatives here: http://python-future.org/compatible_idioms.html#map
I like option 3 (```lmap```) because it is explicit in returning a list whether you're using python 2 or 3.

This could also be re-written as ```[aggregation] if aggregation else []```

Reply:
fwiw, I prefer list comprehension over `filter(None, ...)`

+1 to @nickpell's final suggestion of rewriting with `if aggregation else`